tests/test_example_dags.py

```diff
@@ -37,6 +38,7 @@ def session():
     return get_session()
 
 
+@lru_cache
```

Comment:
Could `from functools import cache` be used instead of `lru_cache`?
as I understand from the [docs](https://docs.python.org/3/library/functools.html#functools.lru_cache), it's a wrapper around `lru_cache` to improve readability.

Reply:
Didn't know about that 💡, thanks!